api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md

```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
 - *"series"* - data fields go to the chart series.
 
 ---
+The **customizeChart**, **customizeSeries**, and **processCell** fields of the **integrationOptions** object are functions. They accept arguments whose structure is described below.
+
+- **customizeChart**
+
+        <!--JavaScript-->customizeChart: function (chartOptions) {
+            // Change chart properties here
+            return chartOptions; // This line is optional
+        }
+        
+    The **chartOptions** object has the following structure:
```

Comment:
The passed object already has values in the following members and  allows you to customize the automatically generated Chart settings

Reply:
And?

Reply:
These members already have values when customizeChart is called while other members of https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxChart/Configuration/ are 'undefined'.
It's up to you how to describe it better.
Currently it is 'The chartOptions object has the following structure ... ' + 'You can also specify any other Chart properties in this object'